lab_1_classify_by_unigrams/main.py

```diff
@@ -20,6 +20,39 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned
     """
+    file = open(text)
```

Comment:
no reading from the file is required.
the text variable is already a string with which you can work further.

Reply:
fixed